dao/application_dao_test.go

```diff
@@ -326,7 +326,8 @@ func TestApplicationNotExists(t *testing.T) {
 }
 
 // TestApplicationSubcollectionListWithOffsetAndLimit tests that SubCollectionList() in application dao returns
-//  correct count value and correct count of returned objects
+//
+//	correct count value and correct count of returned objects
 func TestApplicationSubcollectionListWithOffsetAndLimit(t *testing.T) {
 	testutils.SkipIfNotRunningIntegrationTests(t)
```

Comment:
we have to fix it manually because there are 2 spaces in the second comment line and the linter "fixes" it by adding an extra new empty line but when you remove one of the spaces the linter is happy

Reply:
yea, I have it but I did not push it, thanks for noticing!